tests/core/util/test_streamable.py

```diff
@@ -371,5 +365,11 @@ def test_parse_str(self):
             parse_str(io.BytesIO(b"\x00\x00\x02\x01" + b"a" * 512))
 
 
+def test_json(bt):
+    block = bt.create_genesis_block(test_constants, bytes32([0] * 32), uint64(0))
+    dict_block = block.to_json_dict()
+    assert FullBlock.from_json_dict(dict_block) == block
```

Comment:
Did this need to move?

Reply:
Yes! When executed as a part of the class, it broke because of interactions stemming from `TestStreamable` inheriting from `unittest.TestCase`

Reply:
Oh, right...  it grew a fixture dependency.  Maybe we should convert this from unittest to pytest fully as a pre-PR.

Reply:
I'm happy to write it.